### PR TITLE
Release 2.3.3

### PR DESCRIPTION
**Pull request by YogeshSampalDell**
**Summary**

- [OBSDEF-57542] Support jammy stemcell
- Updated documentation
  - Updated release notes
  - Corrected Broadcom and Github links

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -30,25 +30,19 @@ With ECS Service Broker for VMware Tanzu, you can do the following:
...
-| Software component version | 2.3.2                                                                        |
-| Compatible Ops Manager version(s) | v3.0.x, v2.10.x, v2.9.x                                    |
-| Compatible Tanzu Application Service version(s) | v4.0.x, v3.0.x, v2.13.x, v2.12.x, v2.11.x, v2.10.x, v2.9.x |
-| BOSH stemcell version | Ubuntu Xenial 621                                                            |
-| IaaS support | AWS, Azure, OpenStack, and vSphere                                           |
-| IPsec support? | No                                                                           |
+| Element      | Details                            |
+|:-------------|:-----------------------------------|
+| Tile Version | 2.3.3                              |
+| Release Date | November 25, 2024                  |
```
Comment (YogeshSampalDell) [inline docs/index.md:36]: @twincitiesguy This is just a placeholder date. We'll update after confirmation.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,9 @@ be configured from the PCF Ops Manager.
 
 ## Current Status
 
-Currently, the Tile version 2.3.2 is available for download via the [Pivotal Network product page](https://network.pivotal.io/products/ecs-service-broker/).
+Currently, the Tile version 2.3.3 is available for download via the [Broadcom Support portal](https://support.broadcom.com/group/ecx/productdownloads?subfamily=ECS%20Service%20Broker%20for%20VMware%20Tanzu).
+* This is the last release of the tile, no further versions will be released.
```
Comment (twincitiesguy) [inline README.md]: this needs to be more prominent - recommend bolding and italicizing these 2 lines

File: docs/usage.md [hunk 3]
```diff
@@ -134,14 +134,15 @@ Run `cf delete-service` to delete the service.
 ```
   $ cf delete-service my-bucket
 ```
+-- Delete policy section
```
Comment (twincitiesguy) [inline docs/usage.md]: what is this line for?  looks like a placeholder for some other content

**Review by twincitiesguy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (YogeshSampalDell) [reply README.md]: Updated

File: docs/usage.md (hunk 3 above)
Comment (YogeshSampalDell) [reply docs/usage.md]: Right, removed it.

File: docs/index.md (hunk 1 above)
Comment (YogeshSampalDell) [reply docs/index.md:36]: We are going with the same date i.e 25th Nov

File: docs/release-notes.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
 [Home](index.md) | [Installing](installing.md) | [Using](usage.md) | [Release Notes](release-notes.md) | [Troubleshooting](troubleshooting.md)
 
+## <a id="ver-2-3-3"></a> v2.3.3
+
+**Release Date:** November 25, 2024
+
+In this release:
+
+* Support for Tanzu Application Services version 5.x and 6.x
```
Comment (twincitiesguy) [inline docs/release-notes.md:12]: need to add notices about last relase and end of support here - also bold, italicized, etc.

File: tile.yml [hunk 5]
```diff
@@ -138,11 +138,10 @@ packages:
 # strongly recommend you leave this alone so that your tile always
 # runs with the latest stemcell.
 #
-# stemcell_criteria:
-#   os: ubuntu-trusty
-#   requires_cpi: false
-#   version: '3062'
-
+stemcell_criteria:
```
Comment (twincitiesguy) [inline tile.yml]: according to the comment, if we set this to a specific value, the tile won't run with the latest stemcell - given that it was previously commented, why did we uncomment this?

**Review by twincitiesguy — CHANGES_REQUESTED** [review]

File: docs/release-notes.md (hunk 4 above)
Comment (YogeshSampalDell) [reply docs/release-notes.md:12]: Added.
image

File: tile.yml (hunk 5 above)
Comment (YogeshSampalDell) [reply tile.yml]: Even though the comment says that _"tile always runs with the latest stemcell"_, as per [the code](https://github.com/cf-platform-eng/tile-generator/blame/master/tile_generator/config.py#L311), the tile_generator was still using Xenial as the default stemcell. So to test the broker with Jammy, we had to uncomment this & specify Jammy stemcell here.
I think by latest stemcell, they mean latest available version of the specified stemcell (Xenial, Jammy etc).

Just a couple of days ago, they merged a PR to use Jammy by default. But they haven't officially release new tile_generator version with this change (it's included in a Pre-release version though). 
Ref - https://github.com/cf-platform-eng/tile-generator/releases

I have commented this block & generated a tile using latest _Pre-release_ version of the tile_generator. 
In this tile, tile_generator added below block:
```
stemcell_criteria:
  os: ubuntu-jammy
  requires_cpi: false
  version: '1'
```

I could not test this as I'm facing some issues with the ops-manager now. I'll try to get that working again & test this tile.

I'm even not sure if we should use this version of the tile_generator or not. What do you think ?

Comment (twincitiesguy) [reply tile.yml]: that is a question for the Tanzu team - I guess they are using email now (tanzu-isv-partnerships.pdl@broadcom.com) - let's leave this commented out, and can you write an email to Tanzu with the above summary, and ask them what their recommendation is?  Please copy our team as well.

Comment (YogeshSampalDell) [reply tile.yml]: The tile _generator with jammy as default stemcell is officially released now!
https://github.com/cf-platform-eng/tile-generator/releases/tag/v15.2.0

Comment (twincitiesguy) [reply tile.yml]: in that case, I recommend we revert this to be commented out and verify the default is Jammy.

**Review by twincitiesguy — APPROVED** [review]

File: tile.yml (hunk 5 above)
Comment (YogeshSampalDell) [reply tile.yml]: We've verified this with latest tile_generator.

**Pull request merged**